app/components/post_list/post/body/files/image_file_overlay.tsx

```diff
@@ -35,9 +36,8 @@ const getStyleSheet = (scale: number, th: Theme) => {
 
 const ImageFileOverlay = ({theme, value}: ImageFileOverlayProps) => {
     const dimensions = useWindowDimensions();
-    const scale = dimensions.width / 320;
-    const style = getStyleSheet(scale, theme);
-    return null;
+    const isTablet = useIsTablet();
+    const style = getStyleSheet(isTablet ? dimensions.scale : 1, theme);
```

Comment:
Out of the scope probably, but doesn't this generate a new style on every render?